frontend/src/components/Item/Metadata/ImageEditor.vue

```diff
@@ -10,7 +10,7 @@
             class="ma-2"
             variant="outlined"
             width="288">
-            <v-img :src="imageFormat(item)" contain width="288" height="162" />
+            <v-img :src="imageFormat(item)" :width="288" :aspect-ratio="1.78" />
```

Comment:
I removed both `width` and `contain` because they're not defined on the v-img api (https://vuetifyjs.com/en/api/v-img/) and were throwing type errors. I know the vuetify 3 docs are lacking at the moment and that may just be an error.

The added aspect ratio is to achieve the same dimensions as before (288 / 1.78 ~= 162, which is the old height).

So the display of these images should be the same and be responsive as they were before.

Reply:
That being said, I'll look to putting these into some responsive classes to make them reflow nicely. I'll use width 100% and the existing aspect ratio.

Reply:
Ok, so these are updated to use vuetify's breakpoints to display a different number of images based on the screen size in both ImageEditor and ImageSearch. The aspect ratio that is used to display different types of images is now pulled from a common utility function too. Let me know what you think, happy to tweak this a bit more.